### PR TITLE
Add Support and Organization for Various Kingly_Keys Keyboards and Subfolders

### PR DESCRIPTION
**Pull request by TheRoyalSweatshirt**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by TheRoyalSweatshirt** [comment]
Alright...I guess I give up lol

Tried to start fresh and fix the conflict but I did basically nothing.  I have zero clue what that Conflict is.

**Comment by TheRoyalSweatshirt** [comment]
OH MY GOD I DID IT

**Comment by TheRoyalSweatshirt** [comment]
Can't figure out why Travis is failing. 

But I know it has something to do with that HEAD of the branch I had.

**Comment by fauxpark** [comment]
Ah... you've deleted the LUFA submodule :/

**Comment by TheRoyalSweatshirt** [comment]
> 
> 
> Ah... you've deleted the LUFA submodule :/

Realized this and then read your comment lol

**Comment by drashna** [comment]
This may be a pain to fix.  

the simplest way may be to run:
```
git reset HEAD~5 --hard
``` 

to get back to a good state.  Though, be very careful with this, as you can make things worse.   

And then running `git rebase upstream/master` may be a better way to update things.

File: .gitignore [hunk 1]
```diff
@@ -69,7 +69,15 @@ util/Win_Check_Output.txt
 # things travis sees
 secrets.tar
 id_rsa_*
+<<<<<<< HEAD
```
Comment (fauxpark) [inline .gitignore]: More merge artifacts, please remove these added lines :)

File: keyboards/kingly_keys/little_foot/little_foot.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define SPLITSPACEBASE( \
```
Comment (fauxpark) [inline keyboards/kingly_keys/little_foot/little_foot.h]: I think this should be:

```suggestion
#define LAYOUT_split_space_base( \
```
?

File: keyboards/kingly_keys/little_foot/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Little Foot
+
+![Little_Foot](https://i.imgur.com/M0Usdtg.jpg)
+===
+
+A mighty small keyboard.
+
+Keyboard Maintainer: Garret G. - a.k.a. [The_Royal](https://www.reddit.com/user/The_Royal/) on Reddit, and [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) on GitHub
```
Comment (fauxpark) [inline keyboards/kingly_keys/little_foot/readme.md]:
```suggestion
Keyboard Maintainer: Garret G. - a.k.a. [The_Royal](https://www.reddit.com/user/The_Royal/) on Reddit, and [TheRoyalSweatshirt](https://github.com/TheRoyalSweatshirt) on GitHub  
```

File: keyboards/kingly_keys/romac/keymaps/jarred/keymap.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
+#define _FN1 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[_BASE] = LAYOUT(
+		KC_7, KC_8, KC_9, \
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac/keymaps/jarred/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/kingly_keys/romac/keymaps/stanrc85/config.h [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#undef RGBLED_NUM
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac/keymaps/stanrc85/config.h]: Probably needs a:

```suggestion
#pragma once

#undef RGBLED_NUM
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define FN 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[BASE] = LAYOUT(
+		KC_7, KC_8, KC_9, \
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+      oled_write_P(PSTR("FN\n"), false);
+      break;
+    default:
+      // Or use the write_ln shortcut over adding '\n' to the end of your string
+      oled_write_ln_P(PSTR("Undefined"), false);
+  }
+
+  // Host Keyboard LED Status
+  uint8_t led_usb_state = host_keyboard_leds();
+  oled_write_P(led_usb_state & (1<<USB_LED_NUM_LOCK) ? PSTR("NUMLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK) ? PSTR("NUMLCK ") : PSTR("       "), false);
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  oled_write_P(led_usb_state & (1<<USB_LED_CAPS_LOCK) ? PSTR("CAPLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK) ? PSTR("CAPLCK ") : PSTR("       "), false);
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+  oled_write_P(led_usb_state & (1<<USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```

File: keyboards/kingly_keys/romac_plus/readme.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/readme.md]:
```suggestion
```

File: keyboards/kingly_keys/romac_plus/readme.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+# RoMac+
+
+![RoMac+](https://i.imgur.com/YJ2nCrS.png)
+
+An Upgraded “Plaid” Inspired 12-Key (3x4) Macropad With Fancy Features.
+
+- Keyboard Maintainer: [Garret G.](https://github.com/TheRoyalSweatshirt)
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/readme.md]:
```suggestion
Keyboard Maintainer: [Garret G.](https://github.com/TheRoyalSweatshirt)  
```

File: keyboards/kingly_keys/romac_plus/readme.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+- Hardware Supported: RoMac+ Rev3.0, Pro Micro, Elite-C, Proton C, BlueMicro.
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/readme.md]:
```suggestion
Hardware Supported: RoMac+ Rev3.0, Pro Micro, Elite-C, Proton C, BlueMicro.  
```

File: keyboards/kingly_keys/romac_plus/readme.md [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+- Hardware Availability: Through Online Store [Kingly-Keys](https://kingly-keys.xyz/)  (Stock Varies).
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/readme.md]:
```suggestion
Hardware Availability: Through Online Store [Kingly-Keys](https://kingly-keys.xyz/)  (Stock Varies).
```

File: keyboards/kingly_keys/romac_plus/rules.mk [hunk 14]
```diff
@@ -0,0 +1,49 @@
+MCU = atmega32u4
+F_CPU = 16000000
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/rules.mk]: Lines 2 to 8 can be deleted. QMK provides these as defaults.

File: keyboards/kingly_keys/ropro/readme.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+The RoPro
```
Comment (fauxpark) [inline keyboards/kingly_keys/ropro/readme.md]:
```suggestion
# The RoPro
```

File: keyboards/kingly_keys/ropro/readme.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+===
```
Comment (fauxpark) [inline keyboards/kingly_keys/ropro/readme.md]:
```suggestion
```

File: keyboards/kingly_keys/ropro/rules.mk [hunk 17]
```diff
@@ -0,0 +1,67 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
```
Comment (fauxpark) [inline keyboards/kingly_keys/ropro/rules.mk]: Lines 4 to 38 can be deleted.

File: keyboards/kingly_keys/soap/README.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
+SOAP
```
Comment (fauxpark) [inline keyboards/kingly_keys/soap/README.md]:
```suggestion
# SOAP
```

File: keyboards/kingly_keys/soap/README.md [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+===
```
Comment (fauxpark) [inline keyboards/kingly_keys/soap/README.md]:
```suggestion
```

File: keyboards/kingly_keys/soap/rules.mk [hunk 20]
```diff
@@ -0,0 +1,67 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
```
Comment (fauxpark) [inline keyboards/kingly_keys/soap/rules.mk]: Lines 4 to 38 can be deleted.

**Comment by TheRoyalSweatshirt** [comment]
@noroadsleft I figured. I have zero clue what I did to mess it up so bad. 

Maybe I’ll just keep it all as a personal repo to keep Up with it better until I figure out how in the world to fix it lol.

**Comment by noroadsleft** [comment]
@TheRoyalSweatshirt,

If you're still online in a couple of hours I'll see about setting up a screenshare over Discord and we can see about fixing this up.

**Pull request closed without merging**